packages/docs/configuration/readme.md

```diff
@@ -51,44 +51,19 @@ module.exports = {
 
 [Documentation](https://tailwindcss.com/docs/theme/#screens)
 
-### Colors
-
-```js{3-14}
-module.exports = {
-  theme: {
-    colors: {
-      transparent: 'transparent',
-      white: '#fff',
-      black: '#000',
-      current: 'currentColor',
-    },
-    textColor: theme => theme('colors'),
-    backgroundColor: theme => theme('colors'),
-    fill: theme => theme('colors'),
-    borderColor: theme => theme('colors'),
-    placeholderColor: theme => theme('colors'),
-    stroke: theme => theme('colors'),
-  },
-};
-```
-
-[Documentation](https://tailwindcss.com/docs/theme/#colors)
-
-::: tip
-You should add colors to your project via the `theme.extend` property to not override these defaults.
-:::
-
 ### Layers
 
 ```js{3-9}
 module.exports = {
   theme: {
-    zIndex: {
-      goku: '9000',
-      above: '2',
-      default: '1',
-      under: '-1',
-      limbo: '-999',
+    extend: {
+      zIndex: {
+        goku: '9000',
+        above: '2',
+        default: '1',
+        under: '-1',
```

Comment:
**question** Should we follow tailwind logic and use numerical names for z-indexes ? like `z-1`, `-z-1` ?

Reply:
Good idea, the `generateZIndexes` function could also return the good number: 

```js
console.log(generateZIndexes([1,2]));
// { '1': 1, '0': 0, '2': 2, '3': 3 }

console.log(generateZIndexes([1, 9000]));
// { '0': 0, '1': '1', '2': 2, '8999': 8999, '9000': 9000, '9001': 9001 }
```

This would be more generic.

Any thought on the subject @studiometa/core?

Reply:
@titouanmathis How does it look in addition to the default behaviour ? Do we need to mix array and object?

Reply:
There is no array, only an object with numbers as keys:

```js
function generateZIndexes(indexes) {
  return Object.fromEntries(
    indexes
      .map((value) => [
        [value - 1, value - 1],
        [value, value],
        [value + 1, value + 1],
      ])
      .flat()
  );
}

generateZindexes([1, 9000]);
// { 0: 0, 1: 1, 2: 2, 8999: 8999, 9000: 9000, 9001: 9001 }
```

Reply:
For me, this function is no longer of interest in this way. Maybe we can just add the value `-z-1` and be satisfied with the other default values? @titouanmathis @perruche

Reply:
agree with @antoine4livre, my comment was maybe missleading. I was refering to the default values added to the config

Reply:
Yes, go! 🚀